sdk/test/metrics/sync_metric_storage_test.cc

```diff
@@ -105,6 +105,30 @@ TEST_P(WritableMetricStorageTestFixture, LongSumAggregation)
     expected_total_put_requests = 0;
   }
 
+  // collect one more time.
+  collection_ts    = std::chrono::system_clock::now();
+  count_attributes = 0;
```

Comment:
Is this variable used?

Reply:
Good catch. It should have been used. Have added the validation to check if the collect() gets metrics with all the attributes. In case of Delta temporarily, for simultaneous collections only the first collect() will get the metrics, the subsequent collect() will return entry. Have added that validation too.